<launch>
  <param name="bag_path" value="$(arg bag_path)"/>
  <param name="need_compression" value="$(arg need_compression)"/>
  <rosparam command="load" file="$(find mpl_dataset_toolbox)/config/message_topic.yaml"/>
  <node pkg="mpl_dataset_toolbox" type="bag_splitter" name="bag_splitter" output="screen"/>
</launch>
